index.html

```diff
@@ -1,25 +1,36 @@
 <!DOCTYPE HTML>
 <html lang="en">
     <head>
-        <meta charset="UTF-8">
         <title>OpenVegeMap</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
-        <script src="cordova.js" defer></script>
-        <script src="dist/style.bundle.js" defer></script>
-        <script src="dist/main.bundle.js" defer></script>
-        <link rel="manifest" href="manifest.json">
-        <link rel="icon" href="img/logo_32.png">
-        <link rel="canonical" href="https://openvegemap.netlib.re/">
-        <link rel="preconnect" href="https://overpass-api.de/">
+        <meta charset="utf-8">
         <meta name="description" content="Find vegetarian and vegan restaurants in your city">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
         <meta name="referrer" content="no-referrer">
         <meta name="twitter:creator" content="@Tael67">
-        <meta name="twitter:image" content="https://openvegemap.netlib.re/img/logo_512.png">
+        <meta name="twitter:image" content="https://openvegemap.netlib.re/img/android-chrome-512x512logo_512.png">
         <meta property="og:title" content="OpenVegeMap">
         <meta property="og:url" content="https://openvegemap.netlib.re/">
         <meta property="og:image" content="https://openvegemap.netlib.re/img/banner.png">
         <meta property="og:type" content="website">
         <meta property="og:description" content="Find vegetarian and vegan restaurants in your city">
+        <link rel="manifest" href="manifest.json">
+        <link rel="canonical" href="https://openvegemap.netlib.re/">
+        <link rel="preconnect" href="https://overpass-api.de/">
+
+        <!-- favicons (build with https://realfavicongenerator.net) -->
+        <link rel="apple-touch-icon" sizes="180x180" href="img/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="img/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="img/favicon-16x16.png">
+        <link rel="mask-icon" href="img/safari-pinned-tab.svg" color="#00aba9">
```

Comment:
If I understand correctly, this will make Safari use a teal circle as favicon?

Reply:
That wasn't the intention. I've replaced the svg file. Now it should look like this in the preview of the generator:

![Ekrankopio de 2020-10-20 09-06-27](https://user-images.githubusercontent.com/35647502/96552649-571f1480-12b4-11eb-94c9-7eb2a8ba6a43.png)